Ports/milkytracker/patches/DisplayDevice_SDL.patch

```diff
@@ -0,0 +1,73 @@
+--- MilkyTracker_orig/src/ppui/sdl/DisplayDevice_SDL.cpp	2020-12-09 23:58:14.000000000 +0100
```

Comment:
Shouldn't these files be named `.cpp.patch` instead of just `.patch`?

Reply:
Ah, it doesn't matter. nevermind.

Reply:
I was just using `diff -u <original> <patched>` to compare originals with modified files and it seems like `patch` doesn't care about this line and modifies the correct file